src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -1,27 +1,23 @@
 package edu.kis.powp.jobs2d.drivers;
 
 import edu.kis.powp.jobs2d.Job2dDriver;
-import edu.kis.powp.jobs2d.LoggerDriver;
 import edu.kis.powp.observer.Publisher;
 
 /**
  * Driver manager provides means to setup the driver. It also enables other
  * components and features of the application to react on configuration changes.
  */
 public class DriverManager {
-
-    private Job2dDriver currentDriver = new LoggerDriver();
-
+    private DriversComposite currentDriver = new DriversComposite();
     private final Publisher changePublisher = new Publisher();
 
     /**
      * @param driver Set the driver as current.
      */
     public synchronized void setCurrentDriver(Job2dDriver driver) {
-        currentDriver = driver;
+        currentDriver.addDriver(driver);
```

Comment:
Sorry, but to put it mildly, the solution looks not well thought out and not tested.

Reply:
We were aware of that, but considering the fact that the pull request was marked as in progress we thought that is not the problem. As we were not sure about us understanding the problem, we prefered to check if the solution is acceptable before trying to test it